components/patient-data-sharing/push-ui/src/main/resources/PhenoTips/PushPatient.xml

```diff
@@ -839,6 +859,55 @@
     },
 
     // ---------------------------------------------------------------------
+    // Generate consents section UI
+    // ---------------------------------------------------------------------
+    _populateConsentsList: function() {
+      var _this = this;
```

Comment:
are you talking about ```_populateConsentsList```? The underscore is there because the rest of the code uses underscores everywhere.